Makefile

```diff
@@ -230,24 +230,28 @@ ${ISTIO_ENVOY_DEBUG_PATH}: init
 ${ISTIO_ENVOY_RELEASE_PATH}: init
 
 # Pull depdendencies, based on the checked in Gopkg.lock file.
-# Developers must manually call make depend.update if adding new deps
+# Developers must manually call make depend.ensure if adding new deps
 depend: init | $(ISTIO_OUT)
 
 $(ISTIO_OUT) $(ISTIO_BIN):
 	@mkdir -p $@
 
 # Updates the dependencies and generates a git diff of the vendor files against HEAD.
-depend.diff: depend.update | $(ISTIO_OUT)
+depend.diff: depend.ensure | $(ISTIO_OUT)
 	git diff HEAD --exit-code -- Gopkg.lock vendor > $(ISTIO_OUT)/dep.diff
 
 depend.update.full: depend.cleanlock depend.update
 
 depend.cleanlock:
 	-rm Gopkg.lock
 
-depend.update:
-	@echo "Running dep ensure with DEPARGS=$(DEPARGS)"
-	time dep ensure $(DEPARGS)
+depend.ensure:
+	@echo "Running dep ensure"
+	time dep ensure
+
+depend.update: depend.ensure
+	@echo "Running dep ensure with --update"
+	time dep ensure --update
```

Comment:
it's not great to change the behavior of the target that several people got used to use

it also doesn't let you update 1 package anymore does it ?